### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Juna123727**

### CODE REVIEW

**Review by TechStudent10 — COMMENTED** [review]
I don't get the point of this PR.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## Deploy to heroku
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/Dawnfrosty/Mike-bot)
+
```
Comment (wotta) [inline README.md:3]: I would suggest removing this and closing the PR.
This doesn't add any information for this repo.

```suggestion
```

**Comment by Mostafajami32** [comment]
gh pr checkout 11798

**Comment by MohandMohmoud** [comment]
نعم

**Pull request closed without merging**